portal-api/src/constants/index.js

```
@@ -25,4 +26,6 @@ module.exports = {
   PASSPORT_VALIDATION_RESULTS,
   DATE_FORMATS,
   NUMBER,
+  BANKS,
+  ALL: 'all',
```

Please remove.

---

One of the test `selectAtLeastOneBank › should return an error when new user is checker and a bank is selected` is expecting `ALL` . Should I create new definition and place `ALL` under it.